package.json

```diff
@@ -80,6 +81,7 @@
     "gulp-uglify": "3.0.0",
     "gulp-util": "3.0.8",
     "jsdom": "11.0.0",
+    "marked": "0.3.6",
```

Comment:
Also, the `ReactMarkdown` component [uses the `commonmark` module](https://github.com/rexxars/react-markdown/blob/master/package.json#L57) to render Markdown - might be a better idea to use that instead as a top-level dependency so we only have one Markdown implementation in the project